app/Services/PanelService.php

```
@@ -0,0 +1,866 @@
+        } else {
+            // Jika tidak ada data, tambahkan entri kosong
+            return [
+                'message' => 'Website is not found',
+                'status' => -1,
+            ];
+        }
+
+        $url = $this->baseUrl . '/v2/site?action=GetDirUserINI';
+
+        $requestData = $this->generateRequestData();
+        $requestData['id'] = $id;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$id'. (Unused Code Rules)

(UnusedLocalVariable)